react/utils/blocks.ts

```diff
@@ -0,0 +1,79 @@
+export const getBlockRole = (treePathTail: string) => {
+  const blockRoleMatch = treePathTail.match(/\$(after|around|before)_/)
+
+  if (!blockRoleMatch) {
+    return null
+  }
+
+  return blockRoleMatch[1] as 'after' | 'around' | 'before'
+}
+
+export const getBlockPath = (
+  extensions: RenderContext['extensions'],
+  treePath: string
+): BlockPath | null => {
+  const splittedTreePath = treePath.split('/')
```

Comment:
I think splitted is wrong, it's actually splitTreePath. I think I also made this mistake >_<

Reply:
Oops! Nice catch!